es-space-mgmt/main/default/lwc/reservationDetailForm/__tests__/reservationDetailForm.test.js

```diff
@@ -0,0 +1,244 @@
+                });
+                lighntingComboBoxEl.dispatchEvent(changeCity);
+
+                const lighntingSliderEl = element.shadowRoot.querySelector(
+                    'lightning-slider'
+                );
+                // change Number of people
+                const changeNoOfPeople = new CustomEvent('change', {
+                    detail: { value: 100 }
+                });
+                lighntingSliderEl.dispatchEvent(changeNoOfPeople);
+
+                const lighntingInputEl = element.shadowRoot.querySelector(
+                    'lightning-input'
+                );
+                //change Start Date
+                const changeStartDate = new CustomEvent('change', {
+                    detail: { value: '2/14/2020' }
+                });
+                lighntingInputEl.dispatchEvent(changeStartDate);
+
+                const lighntingRadioGroupEl = element.shadowRoot.querySelector(
+                    'lightning-radio-group'
+                );
+                //change Date Range
+                const changeDateRange = new CustomEvent('change', {
+                    detail: { value: 7 }
+                });
+                lighntingRadioGroupEl.dispatchEvent(changeDateRange);
+            })
+            .then(() => {
+                expect(element.people).toBe(100);
+                const lighntingComboBoxEl = element.shadowRoot.querySelector(
+                    'lightning-combobox'
+                );
+                expect(lighntingComboBoxEl.value).toBe('San Ramon');
+                expect(element.startdate).toBe('2/14/2020');
+                expect(element.totaldays).toBe(7);
+            });
+    });
+
+    it('draft resveration via button click', () => {
+        const element = createElement('c-reservation-detail-form', {
+            is: ReservationDetailForm
+        });
+        element.state = STATE;
+        element.city = CITY;
+        element.people = 50;
+        element.startdate = START_DATE;
+        element.totaldays = 15;
+        document.body.appendChild(element);
+        // Emit mock response with 0 rows
+        getMarketsByStateAdapter.emit(mockMarketsByStateRecords);
+
+        // listen to customerupdate event
+        const handler = jest.fn();
+        element.addEventListener('draftreservation', handler);
+
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise rejects.
+        return Promise.resolve().then(() => {
+            // verify combobox attributes are populated correctly
+            const button = element.shadowRoot.querySelector('lightning-button');
+            button.click();
+            Promise.resolve().then(() => {
+                expect(handler).toHaveBeenCalled();
+                expect(handler.mock.calls[0][0].detail.numberOfPeople).toBe(20);
```

Comment:
Check here for the whole `detail` object, not only `numberOfPeople`.

Reply:
done!